R/ResampleInstances.R

```diff
@@ -38,3 +38,45 @@ instantiateResampleInstance.RepCVDesc = function(desc, size, task = NULL) {
   g = as.factor(rep(seq_len(desc$reps), each = folds))
   makeResampleInstanceInternal(desc, size, train.inds = train.inds, test.inds = test.inds, group = g)
 }
+
+instantiateResampleInstance.FixedCVDesc = function(desc, size, task = NULL) {
+  initial.window = floor(desc$initial.window * size)
+  skip = floor(desc$skip * size)
+  stops  = (seq(size))[initial.window:(size - desc$horizon)]
```

Comment:
`initial.window` could be 0 here. I assume you'd want `initial.window` to be `max(initial.window, 1)` or something.
Also make sure that `initial.window` is <= `(size - desc$horizon)`.